src/components/editMeasure/editor/cqlFunctionAppliernoParams.cql

```diff
@@ -0,0 +1,16 @@
+library T1 version '16.0.000'
```

Comment:
should this be under __mocks__?

Reply:
It should if there wasn't already another copy of the same file. 
Deleted duplicate